js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.js

```diff
@@ -55,170 +71,384 @@ const PreLaunchChecklist = ( props ) => {
 				<Section.Card>
 					<Section.Card.Body>
 						<VerticalGapLayout size="large">
-							<div className="gla-pre-launch-checklist__checkbox_popover">
+							{ ! values.website_live && (
+								<div className="gla-pre-launch-checklist__checkbox">
+									<CheckboxControl
+										{ ...getInputProps( 'website_live' ) }
+									></CheckboxControl>
+									<FlexBlock>
+										<Panel>
+											<PanelBody
+												title={ __(
+													'My store is live and accessible to all users',
+													'google-listings-and-ads'
+												) }
+												initialOpen={ true }
+												onToggle={ getPanelToggleHandler(
+													'website_live',
+													'pre-launch-checklist',
+													'onboarding-policy-check'
+												) }
+											>
+												<PanelRow>
+													{ __(
+														'We use a WordPress.com account to connect your site to the WooCommerce and Google servers. It ensures that requests (e.g. product feed, clicks, sales, etc) from your site are securely and correctly attributed to your store. It enables a connection to your self-hosted site, and provides a common authentication interface across disparate server configurations and architectures.',
+														'google-listings-and-ads'
+													) }
+												</PanelRow>
+												<p>
+													<AppDocumentationLink
+														context="setup-mc-checklist"
+														linkId="check-website-is-live"
+														type="external"
+														href="https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#store-is-live"
+													>
+														{ __(
+															'Learn more about common landing page issues and how to fix them',
+															'google-listings-and-ads'
+														) }
+													</AppDocumentationLink>
+												</p>
+												<Button
+													isPrimary
+													onClick={ () => {
+														setValue(
+															'website_live',
+															true
+														);
+													} }
+												>
+													{ __(
+														'Confirm',
+														'google-listings-and-ads'
+													) }
+												</Button>
+											</PanelBody>
+										</Panel>
+									</FlexBlock>
+								</div>
+							) }
+							{ values.website_live && (
 								<CheckboxControl
 									label={
 										<span className="gla-pre-launch-checklist__checkbox_popover">
 											<span className="checkbox-label">
 												{ __(
-													'My store website is live.',
+													'My store is live and accessible to all users.',
```

Comment:
Thanks for pointing this out. However, we should keep as first person if the check is passed based on the PRD and [Mock link](https://www.figma.com/file/eQ9O4m2flzrcAiDMXkOW0m/Google-Listings-%26-Ads-v1.x?node-id=7450%3A120810). Please correct me if I'm wrong.

Reply:
> However, we should keep as first person if the check is passed based on the PRD and [Mock link](https://www.figma.com/file/eQ9O4m2flzrcAiDMXkOW0m/Google-Listings-%26-Ads-v1.x?node-id=7450%3A120810). Please correct me if I'm wrong.

Yes, it will be the first person wording if the auto-check is passed. However, I meant the spec in Figma mentioned the wording remains in the second person when an item is confirmed (from unchecked status with the 2nd person wording). Either here or the relative processing code fragment of `! values.website_live`, there is no related implementation.

![image](https://user-images.githubusercontent.com/17420811/189572412-85088f32-e5fa-4977-85de-90774e445b8c.png)

Reply:
Thanks for your suggestions. Updated.